packages/ckeditor5-minimap/src/utils.ts

```diff
@@ -88,29 +90,26 @@ export function getPageStyles() {
  * TODO
  *
  * @protected
```

Comment:
```suggestion
 * @internal
```

The same for functions below.